ansible_collections/arista/avd/tests/integration/targets/filter_hide_passwords/tasks/main.yml

```diff
@@ -2,5 +2,5 @@
 - name: Test hide_passwords filter
   ansible.builtin.assert:
     that:
-      - "{{ 'dummy' | arista.avd.hide_passwords(false) == 'dummy' }}"
-      - "{{ 'dummy' | arista.avd.hide_passwords(true) == '<removed>' }}"
+      - "'dummy' | arista.avd.hide_passwords(false) == 'dummy'"
```

Comment:
Unrelated changes, but saw the warnings from ansible-test while troubleshooting other things.